src/connectors/smart-sort/connectSmartSort.ts

```diff
@@ -98,6 +82,7 @@ const connectSmartSort: SmartSortConnector = function connectSmartSort(
             typeof appliedRelevancyStrictness !== 'undefined' &&
             appliedRelevancyStrictness > 0 &&
             appliedRelevancyStrictness <= 100,
```

Comment:
```suggestion
            appliedRelevancyStrictness !== undefined &&
            appliedRelevancyStrictness > 0,
```

I don't see why this could be > 100

Reply:
AFAIK being greater than 100 is invalid. So I just made it explicit about it. Should we loosen it?

Reply:
it's the applied relevancy strictness, the engine would throw the error when you save the settings, so it's impossible to receive this situation, no?